src/devtools/client/debugger/src/actions/pause/history.ts

```diff
@@ -0,0 +1,30 @@
+import { pauseHistoryDecremented, pauseHistoryIncremented } from "../../reducers/pause";
+import { getPauseHistory, getPauseHistoryIndex } from "../../selectors";
+import { seek } from "ui/actions/timeline";
+import { UIThunkAction } from "ui/actions";
+
+export function seekToPreviousPause(): UIThunkAction {
+  return function (dispatch, getState) {
+    const pauseHistory = getPauseHistory(getState());
+    const pauseHistoryIndex = getPauseHistoryIndex(getState());
```

Comment:
I would maybe pull these out into one selector if they are always going to be used together, but that's just me 🤷